<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Volker Krause &lt;vkrause@kde.org&gt;, KovoKs &lt;tomalbers@kde.nl&gt;</author>
 <class>SMTPSettings</class>
 <widget class="QWidget" name="SMTPSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>474</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="smptTab">
      <attribute name="title">
       <string comment="general smtp settings">General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Account Information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="hostLabel">
            <property name="text">
             <string>Outgoing &amp;mail server:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_host</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KLineEdit" name="kcfg_host">
            <property name="trapEnterKeyEvent" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="usernameLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Login:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_userName</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KLineEdit" name="kcfg_userName">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="trapEnterKeyEvent" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>P&amp;assword:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>password</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KPasswordLineEdit" name="password">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="kcfg_storePassword">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Store SMTP password</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="kcfg_requiresAuthentication">
            <property name="text">
             <string>Server &amp;requires authentication</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string comment="advanced smtp settings">Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="encryptionGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Connection Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QStackedWidget" name="checkCapabilitiesStack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="checkCapabilities">
                <property name="text">
                 <string>Auto Detect</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QProgressBar" name="checkCapabilitiesProgress">
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="noAuthPossible">
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>This server does not support authentication</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Encryption:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="encryptHLayout">
              <item>
               <widget class="QRadioButton" name="encryptionNone">
                <property name="text">
                 <string>&amp;None</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="encryptionSsl">
                <property name="text">
                 <string>&amp;SSL/TLS</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="encryptionTls">
                <property name="text">
                 <string>START&amp;TLS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="portLabel">
              <property name="text">
               <string>&amp;Port:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>kcfg_port</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="kcfg_port">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>25</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="authLabel">
              <property name="text">
               <string>Authentication:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="authCombo"/>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QCheckBox" name="kcfg_useProxy">
              <property name="text">
               <string>Connect using the system proxy settings</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>SMTP Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="kcfg_specifyHostname">
            <property name="text">
             <string>Sen&amp;d custom hostname to server</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="hostnameLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Hostna&amp;me:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_localHostname</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="kcfg_localHostname">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="trapEnterKeyEvent" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="kcfg_specifySenderOverwriteAddress">
            <property name="text">
             <string>Use custom sender address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Sender Address:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KLineEdit" name="kcfg_senderOverwriteAddress">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="trapEnterKeyEvent" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="precommandLabel">
            <property name="text">
             <string>Precommand:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="KLineEdit" name="kcfg_precommand">
            <property name="trapEnterKeyEvent" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
   <customwidget>
    <class>KPasswordLineEdit</class>
    <extends>QWidget</extends>
    <header>kpasswordlineedit.h</header>
   </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>kcfg_host</tabstop>
  <tabstop>kcfg_requiresAuthentication</tabstop>
  <tabstop>kcfg_userName</tabstop>
  <tabstop>password</tabstop>
  <tabstop>kcfg_storePassword</tabstop>
  <tabstop>checkCapabilities</tabstop>
  <tabstop>encryptionNone</tabstop>
  <tabstop>encryptionSsl</tabstop>
  <tabstop>encryptionTls</tabstop>
  <tabstop>authCombo</tabstop>
  <tabstop>kcfg_specifyHostname</tabstop>
  <tabstop>kcfg_localHostname</tabstop>
  <tabstop>kcfg_specifySenderOverwriteAddress</tabstop>
  <tabstop>kcfg_senderOverwriteAddress</tabstop>
  <tabstop>kcfg_precommand</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_specifyHostname</sender>
   <signal>toggled(bool)</signal>
   <receiver>hostnameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifyHostname</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_localHostname</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_userName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>usernameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>password</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_storePassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifySenderOverwriteAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifySenderOverwriteAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_senderOverwriteAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
